indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -37,43 +37,97 @@
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
     {% set state = abstract.get_track_reviewing_state(track) %}
-    <span class="i-tag outline {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
-        {{ state.title -}}
-    </span>
+    <div class="state-icon {{ mapping_class[state.name] }} flexrow f-a-center f-j-center"
+         title="{{ mapping_title[state.name] }}">
+        <span
+            {%- if state.name in ['positive', 'conflicting'] %}
+                class="icon-checkmark"
+            {% elif state.name in ['mixed', 'negative'] %}
+                class="icon-cross"
+            {%- endif -%}
+        ></span>
+    </div>
```

Comment:
We have the following track states: `positive`, `negative`, `conflicting`, `mixed`.

Based on the meeting we had, we decided on the following:
* `positive`: :heavy_check_mark:  (in green)
* `negative`: :x: 
* `conflicting`: :heavy_check_mark: (in purple)
* `mixed`: ~ (in yellow)

Is that the final decision?
In this case we need to find an icon for `~` (which icomoon doesn't unfortunately have), or apply some custom CSS to the character `~` to make it look like the other icons.